spec/lib/rs_api/hiscores/player_experience_spec.rb

```diff
@@ -46,20 +47,11 @@ class PlayerExperienceTest
           end
         end
 
-        it 'formats experience' do
-          VCR.use_cassette('player_experience__successful', erb: { player_name: }) do
-            player_xp = player.loaded_xp
-
-            # confirm experience value is formatted with .delimited
-            expect(player_xp[0][2]).to include(',').at_least(:once)
-          end
-        end
-
```

Comment:
While in test env, `display` will return `Text::Table`. Could probably keep this test and access an entry in one of the rows of the table